composer.json

```diff
@@ -40,7 +40,8 @@
         "phpstan/phpstan": "^0.12.44",
         "phpstan/phpstan-phpunit": "*",
         "phpstan/phpstan-strict-rules": "^0.12.5",
-        "phpunit/phpunit": "^7.0 || ^8.0"
+        "phpunit/phpunit": "^7.0 || ^8.0",
+        "overtrue/phplint": "*"
```

Comment:
Since phplint is only needed for PHP 5.6 there isn't a real reason to have as part of the composer.json.